packages/attribute-registry-process/test/mockUtils.ts

```diff
@@ -0,0 +1,14 @@
+import { Tenant, generateId } from "pagopa-interop-models";
+
+export const getMockTenant = (): Tenant => ({
+  name: "tenant_Name",
+  id: generateId(),
+  createdAt: new Date(),
+  attributes: [],
+  externalId: {
+    value: "1234",
+    origin: "IPA",
+  },
+  features: [],
+  mails: [],
+});
```

Comment:
This could be deleted since we already have a common tenant mock function `getTenantMock` in `pagopa-interop-commons-test`

Reply:
I deleted it and used the commons export where it was used